main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```diff
@@ -402,17 +404,16 @@ object Monomorpher {
     case LoweredAst.TypeParam(name, sym, loc) => MonoAst.TypeParam(name, sym, loc)
   }
 
-  /**
-    * Converts the given effect op spec. Effect operations are monomorphic -
-    * they have no variables - so the substitution can be empty.
-    */
-  private def visitEffectOpSpec(spec: LoweredAst.Spec, subst: StrictSubstitution): MonoAst.Spec = spec match {
+  /** Converts the given effect op spec. */
+  private def visitEffectOpSpec(spec: LoweredAst.Spec)(implicit root: LoweredAst.Root, flix: Flix): MonoAst.Spec = spec match {
     case LoweredAst.Spec(doc, ann, mod, _, fparams0, declaredScheme, retTpe, eff, _) =>
+      // Effect operations are monomorphic - they have no variables.
+      // The substitution can be left empty.
       val fparams = fparams0.map {
         case LoweredAst.FormalParam(sym, fparamMod, tpe, src, loc) =>
-          MonoAst.FormalParam(sym, fparamMod, subst(tpe), src, loc)
+          MonoAst.FormalParam(sym, fparamMod, StrictSubstitution.empty(tpe), src, loc)
       }
-      MonoAst.Spec(doc, ann, mod, fparams, declaredScheme.base, subst(retTpe), subst(eff))
+      MonoAst.Spec(doc, ann, mod, fparams, declaredScheme.base, StrictSubstitution.empty(retTpe), StrictSubstitution.empty(eff))
```

Comment:
Is this a behavior change?

Reply:
no, StrictSubstitution.empty was just passed as a parameter since it was annoying to create a new empty before